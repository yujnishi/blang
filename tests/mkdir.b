NULL 0;

main() {
    auto dname "mkdir.tmp";

    mkdir(dname,493);
    if ( !fork() ) execl("/usr/bin/stat","stat","-c","%A %n",dname,NULL);
    wait();
}
