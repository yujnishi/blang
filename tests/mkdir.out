drwxr-xr-x mkdir.tmp
